### PR TITLE
ci: fix the  snyk policy to  correctly  ignore the  MPL  license issue.

### LINKED ISSUE #12930: snyk  reports dependencies with  MPL license as vulnerabilities
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**

snyk runs after merges to master and release branches report medium severity issue warnings due to dependency projects using the MPL license. 
 
Example: https://github.com/rook/rook/actions/runs/6223395202/job/16889257921 

**Expected behavior:**
snyk  should pass without warnings and errors

**How to reproduce it (minimal and precise):**

* get a  PR merged to the master branch or a  release branch of rook.
* look for corresponding workflow runs https://github.com/rook/rook/actions?query=branch%3Amaster

**Environment**:
*  environment where this was observed: github CI 
 OS (e.g. from /etc/os-release): any
* Cloud provider or hardware configuration: any
* Rook version: current master branch
* Storage backend version ): any

* Kubernetes cluster type : any/irrelevant 
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/latest-release/Troubleshooting/ceph-toolbox/#interactive-toolbox)): irrelevant

### PR DESCRIPTION
**Pull request by obnoxxx**
**Description of your changes:**

after  https://github.com/rook/rook/pull/12932 has been merged, we continue to see MPL issues reported by the snyk CI runs:

https://github.com/rook/rook/actions/runs/6302663608/job/17110310884

This updates to the `.snyk` policy file to correctly ignore the MPL license issue by using the  the proper snyk ID for this issue

According to the doc
https://docs.snyk.io/manage-risk/policies/the-.snyk-file,

the correct ID is `snyk:lic:npm:symbol:MPL-2.0`

The update to the file has been performed with the `snyk ignore` command to ensure validity of the policy file.

**Which issue is resolved by this Pull Request:**
Resolves #12930 

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [x] Unit tests have been added, if necessary.
- [x] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by obnoxxx** [comment]
note to reviewers: this fix can not be verified pre-merge. Only the post-merge ci snyk run would show whether it worked.

File: .snyk [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
-  MPL-2.0:
-    - reason: "Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license"
+  'snyk:lic:npm:symbol:MPL-2.0':
+    - '*':
+        reason: >-
+          Mozilla Public License 2.0 is compatible with Rook's Apache 2.0
+          license
+        expires: 2023-10-26T10:38:56.640Z
```
Comment (travisn) [inline .snyk:9]: Can we remove the `expires`?

File: .snyk [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+        created: 2023-09-26T10:38:56.645Z
```
Comment (travisn) [inline .snyk:10]: Can we remove `created`? Git history is sufficient.

File: .snyk [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
-  MPL-2.0:
-    - reason: "Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license"
+  'snyk:lic:npm:symbol:MPL-2.0':
+    - '*':
+        reason: >-
```
Comment (travisn) [inline .snyk:6]: nit: could we consolidate to a single line for simplicity?
```suggestion
        reason: Mozilla Public License 2.0 is compatible with Rook's Apache 2.0 license
```

File: .snyk [hunk 4]
```diff
@@ -1,3 +1,12 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
```
Comment (travisn) [inline .snyk:2]: If we can remove expiry date, no need for this comment

**Review by travisn — CHANGES_REQUESTED** [review]

File: .snyk (hunk 4 above)
Comment (obnoxxx) [reply .snyk]: let me try ...  done: removed expiry and comment line

**Review by travisn — APPROVED** [review]

**Pull request merged**